isisd/isis_redist.c

```diff
@@ -374,9 +374,9 @@ static void isis_redist_update_zebra_subscriptions(struct isis *isis)
 			afi_t afi = afi_for_redist_protocol(protocol);
 
 			if (do_subscribe[protocol][type])
-				isis_zebra_redistribute_set(afi, type);
+				isis_zebra_redistribute_set(afi, type, isis->vrf_id);
```

Comment:
Can we run clang-format on this commit?

Reply:
yes sure, I apologize for this oversight.